knowit/__main__.py

```diff
@@ -3,19 +3,22 @@
 
 import json
 import logging
-import os
 import sys
 from argparse import ArgumentParser
 
 from babelfish.language import Language
+
```

Comment:
For the imports, knowit is following PEP 8 style guide:

> Imports should be grouped in the following order:
> - standard library imports
> - related third party imports
> - local application/library specific imports
> You should put a blank line between each group of imports.

There's no need to have a blank line between each third party import.